docs/languages/csharp/delegates-events.md

```diff
@@ -2,4 +2,42 @@
 
 > **Note**
 > 
-> This topic hasn’t been written yet! You can track the status of this issue through our public GitHub issue tracker. Learn how you can contribute on GitHub.
+> This topic will be covered under the following articles:
+
+<style type="text/css">
+ol {
+    list-style-type: upper-roman;
+}
+</style>
+1. [Overview of Delegates](delegates-overview.md)
+
+    This article covers an overview of delegates.
+
+2. [System.Delegate and the `delegate` keyword](delegate-class.md)
+
+    This article covers the classes in the .NET Framework that support delegates    and how that maps to the `delegate` keyword.
```

Comment:
".NET Framework" ---> .NET Core?

This I'm actually unsure of.  I think it's safe to say ".NET Core" here, but this could end up generalizing across .NET platforms.